keyboards/jels/jels88/config.h

```diff
@@ -0,0 +1,61 @@
+/* Copyright 2021 Joah Nelson (Jels)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation,   either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor paramater */
+#define VENDOR_ID       0x006a   // Jels (J)
+#define PRODUCT_ID      0x0088  // Jels 88
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Jels
+#define PRODUCT         Jels88
+
+/* Define Matrix size */
+#define MATRIX_ROWS 12
+#define MATRIX_COLS 9
+
+/*Define Matrix Pins */
+#define MATRIX_ROW_PINS \
+    { B3, B2, D1, D0, E6, B0, F0, F1, B5, B4, D7, D6 }
+#define MATRIX_COL_PINS \
+    { C7, C6, F7, F6, F5, F4, B1, D2, D3 }
+
+/* ws2812b RGB LEDs */
+#define RGB_DI_PIN B6
+
+#ifdef RGBLIGHT_ENABLE
+#    define RGBLED_NUM 20  // Number of LEDs
+#    define RGBLIGHT_SLEEP
+#    define RGBLIGHT_ANIMATIONS
```

Comment:
Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```